res/skins/Deere/style.qss

```diff
@@ -212,16 +215,17 @@ QTableView::indicator:unchecked {
 QPushButton#LibraryPreviewButton {
   width: 23px;
   height: 12px;
-  background: transparent;
   border: none;
 }
 
 QPushButton#LibraryPreviewButton:!checked {
   image: url(skin:/image/style_library_preview_play.png);
+  background-color: #1A1A1A;
 }
 
 QPushButton#LibraryPreviewButton:checked {
   image: url(skin:/image/style_library_preview_pause.png);
+  background-color: #006596;
```

Comment:
I suggest 

"228 background-color: #006596;"

then It looks like this:
![new_fix](https://user-images.githubusercontent.com/3403218/32703691-bd2edd36-c7f9-11e7-8c86-968736046cf2.png)

which is (in my opignion) better than

![with_fix](https://user-images.githubusercontent.com/3403218/32703699-d2cc9ade-c7f9-11e7-81b3-54b3d2699585.png)

Reply:
Blue might be better if Preview deck is not visible and if you scroll around a lot while previewing.
Dark grey looks better but breaks the highlighted line, depending on how columns are sorted.
I don't mind..

Reply:
@Be-ing What do you think?

Reply:
I think the background for a deck playing in the preview deck should be different from both the highlight background color and the unselected background color.

Reply:
Actually, I like it how it is. It jumps out a little bit, but not too much.